EasyPost.Integration/TestUtils.cs

```diff
@@ -0,0 +1,174 @@
+using System.Runtime.CompilerServices;
```

Comment:
question: can we not reuse the config from the unit tests here to avoid duplicating this?

Reply:
Unfortunately not, because of how this file looks up the location of cassettes. If used, cassettes for integration tests would be stored in the cassettes folder under Tests rather than Integrations

Reply:
I can explore making a location-agnostic version of test utils in the future.

Reply:
Cool, I'd love to revisit if it's possible. Dynamically supplying the cassette location so we can keep the other same config pieces DRY would be good.